packages/react-native-codegen/src/parsers/typescript/components/props.js

```diff
@@ -9,38 +9,22 @@
  */
 
 'use strict';
-const {getSchemaInfo, getTypeAnnotation} = require('./componentsUtils.js');
+
+const {buildPropSchema} = require('../../parsers-commons');
```

Comment:
For passing as params
```suggestion

const {getSchemaInfo, getTypeAnnotation} = require('./componentsUtils.js');
const {buildPropSchema} = require('../../parsers-commons');
```